.github/workflows/pr-lint-test-js.yml

```
@@ -23,7 +23,7 @@ jobs:
               uses: ./.github/actions/setup-woocommerce-monorepo
 
             - name: Lint
-              run: pnpm run -r --filter='woocommerce/client/admin...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api' --color lint
+              run: pnpm run -r --filter='tools/release-posts' --filter='woocommerce/client/admin...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api' --color lint
```

```suggestion
              run: pnpm run -r --filter='release-posts' --filter='woocommerce/client/admin...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api' --color lint
```

I'm not sure that the filter you originally suggested actually matches. If I ran it without the others locally, I got an error that there weren't any matches. This updated one matches and runs the lint script that was added to the project.

---

Ah wow. thanks for spotting. I'll commit your suggestion and check the CI